roles/tarzan/tasks/dependencies.yml

```diff
@@ -114,6 +52,34 @@
     chdir: "{{ openresty_dest}}/openresty-{{ openresty_version }}" 
     creates: "{{ openresty_dest }}/{{ openresty_version }}/bin/resty"
 
+- name: create luarocks destination directory
+  file: path={{ luarocks_dest }} state=directory mode=g+rwXs
+
+- name: download luarocks archive
+  get_url:
+    url: "{{ luarocks_url }}"
+    dest: "{{ luarocks_dest }}/{{ luarocks_file }}"
+
+- name: unpack luarocks to destination
+  unarchive: 
+    src: "{{ luarocks_dest }}/{{ luarocks_file }}" 
+    dest: "{{ luarocks_dest }}"
+    copy: no 
+    creates: "{{ luarocks_dest }}/{{ luarocks_version }}/bin/luarocks"
+    mode: g+rwX,o+rX
+
+- name: configure luarocks
+  shell: "./configure --lua-suffix=jit --with-lua={{ openresty_dest }}/{{ openresty_version }}/luajit --with-lua-include={{ openresty_dest }}/{{ openresty_version }}/luajit/include/luajit-2.1 --prefix={{ luarocks_dest }}/{{ luarocks_version }}"
```

Comment:
Is it ok to have version *(2.1)* hard code here i.e. in `--with-lua-include={{ openresty_dest }}/{{ openresty_version }}/luajit/include/luajit-2.1`

Reply:
Maybe could have a place holder variable `luajit__include_version` *(or something)* if it is not exactly same as `luajit_version` in `defaults/main.yaml`

Reply:
I was wondering about that as well, but I don't see a reliable way to figure out the luajit version bundled with openresty. I could add a variable `openresty_bundled_luajit_version` (or similar) to where the other openresty-variables are defined, so that one at least has a reminder in the vicinity to check for that when updating openresty

Reply:
ah ha.. I see your point, maybe if this path `{{ openresty_dest }}/{{ openresty_version }}/luajit/include` is going to be standard i.e. `laujit` bundled with `openresty` is always in that path, then maybe we could deduce the version like

```
- name: Grab luajit version number
  shell: "ls {{ openresty_dest }}/{{ openresty_version }}/luajit/include | grep luajit-[0-9a-z\.]*"
  register: openresty_bundled_luajit_version
```

Reply:
or we just replace it with `*` and let the shell take care of it
```
shell: "./configure --lua-suffix=jit --with-lua={{ openresty_dest }}/{{ openresty_version }}/luajit --with-lua-include={{ openresty_dest }}/{{ openresty_version }}/luajit/include/luajit-* --prefix={{ luarocks_dest }}/{{ luarocks_version }}"
```

Reply:
Sure, I was just being extra paranoid 😜